wss.bat

```diff
@@ -1,41 +1,54 @@
 REM Run whitesource for components which need versioning
+@echo off
+setlocal
 
+if not defined WHITESOURCE_API_KEY (
+    echo == No WHITESOURCE_API_KEY is set. Skipping WhiteSource scan
+    exit /b 0
+)
 SET SCAN_DIRECTORIES="%cd%"
 
-IF "%PRODUCT_NAME%"=="" SET PRODUCT_NAME="Snowflake_Drivers_Connectors"
-
-SET PROJECT_NAME="Dot-Net-Connector"
+SET PRODUCT_NAME=DotNETDriver
+SET PROJECT_NAME=DotNETDriver
 
-SET DATE="%date%"
+REM Format MM-DD-YYYY
+SET CURRENT_DATE=%date:~4,2%-%date:~7,2%-%date:~10,4%
 
 curl -LJO https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
 
 SET WSS_CONFIG="wss-net.config"
+COPY %WSS_CONFIG%.templ %WSS_CONFIG%
 
 java -jar wss-unified-agent.jar -apiKey %WHITESOURCE_API_KEY%^
    -c %WSS_CONFIG%^
    -project %PROJECT_NAME%^
    -product %PRODUCT_NAME%^
-   -projectVersion %DATE%^
    -d %SCAN_DIRECTORIES%^
    -wss.url https://saas.whitesourcesoftware.com/agent^
    -offline true
+IF %ERRORLEVEL% NEQ 0 (
+    echo == failed to run WSS for %PRODUCT_NAME%_%PROJECT_NAME% in offline mode
+    exit /b 1
+)
 
 java -jar wss-unified-agent.jar -apiKey %WHITESOURCE_API_KEY%^
     -c %WSS_CONFIG%^
-    -project %PROJECT_NAME%^
     -product %PRODUCT_NAME%^
+    -project %PROJECT_NAME%^
     -projectVersion baseline^
     -requestFiles whitesource\update-request.txt^
     -wss.url https://saas.whitesourcesoftware.com/agent
 
 IF %ERRORLEVEL% NEQ 0 (

```

Comment:
@sfc-gh-nsfard , I don't think this check works. The above scan with baseline will be success at all times, so the date based versioning won't work. Can you please confirm?

If so, my latest update won't work too, because it always sends the update twice for both baseline and the date version.